src/DynamicRupture/Output/ReceiverBasedOutput.cpp

```diff
@@ -433,25 +449,26 @@ real ReceiverOutput::computeRuptureVelocity(Eigen::Matrix<real, 2, 2>& jacobiT2d
 
     auto* rt = getCellData(local, drDescr->ruptureTime);
     for (size_t jBndGP = 0; jBndGP < misc::NumBoundaryGaussPoints; ++jBndGP) {
-      const real chi = chiTau2dPoints(jBndGP, 0);
-      const real tau = chiTau2dPoints(jBndGP, 1);
+      const real chi = seissol::multisim::multisimTranspose(chiTau2dPoints, 0, jBndGP);
```

Comment:
`seissol::multisim::multisimTranspose(chiTau2dPoints, jBndGP, 0);`; same in the line below. (unless I missed something here and it's transposed in the non-fused case)